sunpy/spectra/tests/test_spectrogram.py

```diff
@@ -14,6 +14,16 @@
     Spectrogram, LinearTimeSpectrogram, _LinearView
 )
 
+from sunpy.spectra.spectrum import Spectrum
+
+def test_spectrum():
+    #The idea is to create two 1-D arrays, data and freq_axis,
+    #Pass them to the Spectrum object and check for equality of dimensions accordingly.
+    data = np.linspace(0, 100, 100)
+    freq = np.linspace(1, 100, 100)
+    spec = Spectrum(data,freq)
+    if not (np.shape(spec.data) == np.shape(spec.freq_axis)):
+        raise ValueError
```

Comment:
But the docs say that , Unit tests are written in the files in the test folder. And Should I raise the error wherever Spectrum objects is declared? And shouldn't I raise the error in Spectrum.py, instead of Spectrogram.py, since that is where the object declaration is made.
P.S. I made the changes in the Spectrum.py file and it does correctly raise a ValueError when data and freq_axis dimensions are incorrect. 
How Do I proceed from here. Thanks.